public/class-wa-external-header-v2-public.php

```diff
@@ -52,15 +52,15 @@ class Wa_External_Header_V2_Public
 
     private static $instance;
 
-    private $white_album_css_namespace,
-        $afubar,
-        $header,
-        $footer,
-        $user_config,
-        $start_tag,
-        $end_tag,
-        $banners,
-        $head;
+    private $white_album_css_namespace;
+    private $afubar;
```

Comment:
Skal være opdelt ifg. PSR-2: http://www.php-fig.org/psr/psr-2/

`There MUST NOT be more than one property declared per statement.`